src/Interpolator.js

```diff
@@ -213,7 +218,10 @@ class Interpolator {
       }
 
       if (doReduce) {
-        value = formatters.reduce((v, f) => this.format(v, f, options.lng, options), value.trim());
+        value = formatters.reduce(
+          (v, f, k) => this.format(v, f, options.lng, { ...options, interpolationkey: k }),
```

Comment:
Isn't k not just the index in the array? Dies this work?

Reply:
No, test case revealed it. It didn't cause an issue, but it didn't do what was expected.  This has been changed to properly pass the key name to the format function.